### DISCUSSION TITLE
xrightarrow looks good on Firefox but bad on Edge, Chrome

### DISCUSSION THREAD

**Discussion by vochicong** [quarto-dev/quarto-cli#2882] [Q&A]
I've created [a math web page](https://vochicong.github.io/xstk/30-est-method.html#chu%E1%BA%A9n-t%C3%ADnh-c%E1%BB%A7a-mle) using Quarto.
The `\xrightarrow`

$X_n \xrightarrow[n\to\infty]{P} X$

looks nice on Firefox, but appears too short on Edge or Chrome.
Do you know how to make it better?

Firefox: <img width="123" alt="image" src="https://user-images.githubusercontent.com/123111/196098328-afaa945f-35e4-486d-b517-13f685edd192.png">

Edge or Chrome: <img width="129" alt="image" src="https://user-images.githubusercontent.com/123111/196098361-ba83cbbe-b2ad-427d-80b1-438bb9d58865.png">

**Comment by cscheid** [comment]
I see this on Chrome:

<img width="657" alt="image" src="https://user-images.githubusercontent.com/285675/196210908-7ed1dc9b-9e82-44e7-b345-75c67afe72bf.png">

**Reply by vochicong** [reply]
@cscheid Thank you for looking. Your Chrome correctly shows the math! What OS are you using?

My environment:

- Windows 11 Pro 22H2
- GoogleChrome v106.0.5249.119
- MicrosoftEdge v106.0.1370.47
- Firefox v105.0.3

**Reply by cscheid** [reply]
I'm on the latest macOS. I don't think this is a quarto issue; I suspect this is a MathJax problem. Have you tried that specific equation on a standalone MathJax install?

**Reply by vochicong** [reply]
No, I haven't tested a standalone MathJax.
